### ISSUE TITLE
Get air quality methods should have option to return different data formats

### ISSUE THREAD

**Issue by Milind220** [Milind220/Ozone#1]
There should be an option to return data in CSV or Excel format, in case that is desired. Not everyone will want to use Pandas data frames for data analysis.

**Comment by ubongab** [comment]
I'll be happy to work on this feature.

**Comment by Milind220** [comment]
@ubongab That'd be great! I'll take a look at your other PR in a while. 

Thanks a lot

**Comment by ubongab** [comment]
I have added `.json `file and `.csv` file so far but the `.xlsx` requires an excel writer: `openpyxl`.
Is it something that you can add to the global package dependencies to enable this work automatically?

**Comment by Milind220** [comment]
@ubongab Yes, that's absolutely possible. I've used openpyxl before, and it's fairly reliable as third party packages go. 

You can go ahead and add the .xlsx file writing feature, and I'll add openpyxl to the dependencies once that's done.
